api/routes.ts

```diff
@@ -5,17 +5,15 @@ import healthCheck from './healthCheck'
 import getappInsightsInstrumentationKey from './monitoring-tools'
 import organisationRouter from './organisation'
 import pbaAccounts from './pbaAccounts'
-import pbaRouter from './updatePba'
 import stateRouter from './states'
+import pbaRouter from './updatePba'
 
 const router = express.Router({ mergeParams: true })
 // open routes
 router.use('/environment', environment)
 
 router.use(auth.attach)
 
-router.use('/logout', auth.logout)
```

Comment:
bit confused about the removal of this?

Reply:
Its in the application.ts already. Hence, removed it